msicreator/create_installer.py

```diff
@@ -0,0 +1,62 @@
+import os, sys, subprocess
+from pathlib import Path
+from shutil import copytree, copy2
+import json
+import createmsi
+
+print('First arg:', sys.argv[1])
+build_dir = sys.argv[1]
+src_dir = sys.argv[2]
+work_dir = build_dir + '/workdir'
+project_path = work_dir + '/msicreator/libo-project'
+
+def prepare_project_dir():
+    instdir = build_dir + '/instdir'
+    try:
+        print('Copying build files...')
+        copytree(instdir, project_path + '/main', dirs_exist_ok=True)
+        print('Copying build files finished.')
+    except FileExistsError as err:
+        print(err)
+
+def create_creator_json():
+    print('Generating json file for MSI Creator...')
+    lo_dictionary = {
+        "upgrade_guid": "6f05ed48-a735-4155-ab60-e4cc98455262",
+        "version": "7.4.2",
```

Comment:
Since you wrote this information in a JSON file in `configure.ac` above, you should read that file and get the version number from that instead of hardcoding.

Reply:
During development, I asked @zonnebloempje to work on reading the content of a JSON file and print it to the terminal as preparation for updating the values here. She did and has the code for updating the values with the ones from the JSON file. I left the hardcoded values here so she could update them with what she has locally when this PR is merged.

Reply:
Ok, good.